utils/epinow2/functions.py

```diff
@@ -113,6 +117,23 @@ def validate_args(
         A dictionary of sanitized arguments.
     """
     args_dict = {}
+    # Split up input string of task_exclusions and validate
+    if task_exclusions is not None:
+        task_pairs = task_exclusions.split(",")
```

Comment:
These `splits` will fail with an `IndexError` if the string is misformed -- can we wrap this whole block in a try/except to handle that case and add a few tests to check that they fail gracefully?